### PR TITLE
ARTEMIS-1341 fix getBytes

### PR DESCRIPTION
**Pull request by stanlyDoge**
https://issues.apache.org/jira/browse/ARTEMIS-1341

### CODE REVIEW

File: tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/jms/jms2client/BodyTest.java [hunk 1]
```diff
@@ -59,13 +58,19 @@ public void testBodyConversion() throws Throwable {
          producer.send(bytesMessage);
 
          Message msg = cons.receiveNoWait();
-         assertNotNull(msg);
```
Comment (michaelandrepearce) [inline tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/jms/jms2client/BodyTest.java]: this should be kept.

**Comment by michaelandrepearce** [comment]
Behaviour should be the same, which ever JMS client they use, I'm not in favour of behaviours being different, this gives a bad experience to users if one client behaves one way and another behaves another.

**Comment by stanlyDoge** [comment]
> Behaviour should be the same
True, there was little misunderstanding. I am working at fix.

**Comment by stanlyDoge** [comment]
This is just test fix. There are some issues with clients. I'll get back to it later.

**Comment by jdanekrh** [comment]
> Likewise i don't understand why OpenWire is skipped

Skipping OpenWire is appropriate here, as [`Message#getBody`](https://docs.oracle.com/javaee/7/api/javax/jms/Message.html#getBody-java.lang.Class-) was added in JMS 2.0, and activemq-client implements only the 1.1 version of the JMS spec.

**Comment by michaelandrepearce** [comment]
@jdanekrh cheers, makes perfect and very obvious sense. *slaps hand against forehead and smashes it on table* it’s been a long few days.

**Comment by michaelandrepearce** [comment]
@stanlyDoge is this ready? If so could you squash the commits into one?

**Comment by stanlyDoge** [comment]
@michaelandrepearce Yes, it should be ready.

**Comment by michaelandrepearce** [comment]
@stanlyDoge unfortunately it seems there is a build failure of this PR, could you look at it?

**Comment by stanlyDoge** [comment]
Oops with squashing. Should be ok now, let's wait for check.

**Comment by clebertsuconic** [comment]
@stanlyDoge I believe this is breaking the compatiblity testsuite I just added...

I'm glad I did :)

**Comment by clebertsuconic** [comment]
@stanlyDoge actually.. my bad.. I had something wrong...

At least I'm watching it ! :)

**Comment by stanlyDoge** [comment]
@clebertsuconic double check is better than no check :) Thank you.

**Pull request closed without merging**